pandas/core/series.py

```diff
@@ -2292,10 +2303,23 @@ def combine(self, other, func, fill_value=None):
         1    2
         dtype: int64
 
-        See Also
-        --------
-        Series.combine_first : Combine Series values, choosing the calling
-            Series's values first
+        >>> s2 = pd.Series([0, 3, 4])
+        >>> s1.combine(s2, lambda x1, x2: x1 if x1 > x2 else x2)
+        0    1
+        1    3
+        2    4
+        dtype: int64
+
+        When fill_value is given:-
+        >>> s1.combine(s2, lambda x1, x2: x1 if x1 > x2 else x2,
+        ...            fill_value = 787)
+        0      1
+        1      3
+        2    787
+        dtype: int64
+
+        If `func` doesn't get a value from either of the two Series,
```

Comment:
This is not necessarily true:

```
In [3]: s1
Out[3]:
0    1
1    2
dtype: int64

In [4]: s2
Out[4]:
0    0
1    3
2    4
dtype: int64

# 100 > 4 so we get x1(100)
In [5]: s1.combine(s2, lambda x1, x2: x1 if x1 > x2 else x2, fill_value=100)
Out[5]:
0      1
1      3
2    100
dtype: int64

# 100 !< 4 so we get x2 (4)
In [6]: s1.combine(s2, lambda x1, x2: x1 if x1 < x2 else x2, fill_value=100)
Out[6]:
0    0
1    2
2    4
dtype: int64
```
`fill_value` is the _default comparison value_ in the function if a value is missing in the series not what gets filled in the result.